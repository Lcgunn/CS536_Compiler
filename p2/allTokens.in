!! This is a comment .j03-2~ \n
2024
=
~=
$ This is also a comment <3
_a0
Ao = 0
if delish
"very bad string \
abc