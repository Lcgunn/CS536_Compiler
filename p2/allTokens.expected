2024
=
~=
_a0
Ao
=
0
if
delish
abc
